packages/flutter/lib/src/material/text_field.dart

```diff
@@ -277,53 +280,70 @@ class TextField extends StatefulWidget {
   @override
   void debugFillProperties(DiagnosticPropertiesBuilder properties) {
     super.debugFillProperties(properties);
-    properties.add(new DiagnosticsProperty<TextEditingController>('controller', controller, defaultValue: null));
-    properties.add(new DiagnosticsProperty<FocusNode>('focusNode', focusNode, defaultValue: null));
-    properties.add(new DiagnosticsProperty<InputDecoration>('decoration', decoration));
-    properties.add(new DiagnosticsProperty<TextInputType>('keyboardType', keyboardType, defaultValue: TextInputType.text));
-    properties.add(new DiagnosticsProperty<TextStyle>('style', style, defaultValue: null));
-    properties.add(new DiagnosticsProperty<bool>('autofocus', autofocus, defaultValue: false));
-    properties.add(new DiagnosticsProperty<bool>('obscureText', obscureText, defaultValue: false));
-    properties.add(new DiagnosticsProperty<bool>('autocorrect', autocorrect, defaultValue: false));
+    properties.add(new DiagnosticsProperty<TextEditingController>(
+        'controller', controller,
+        defaultValue: null));
+    properties.add(new DiagnosticsProperty<FocusNode>('focusNode', focusNode,
+        defaultValue: null));
+    properties.add(
+        new DiagnosticsProperty<InputDecoration>('decoration', decoration));
+    properties.add(new DiagnosticsProperty<TextInputType>(
+        'keyboardType', keyboardType,
+        defaultValue: TextInputType.text));
+    properties.add(
+        new DiagnosticsProperty<TextStyle>('style', style, defaultValue: null));
+    properties.add(new DiagnosticsProperty<bool>('autofocus', autofocus,
+        defaultValue: false));
+    properties.add(new DiagnosticsProperty<bool>('obscureText', obscureText,
+        defaultValue: false));
+    properties.add(new DiagnosticsProperty<bool>('autocorrect', autocorrect,
+        defaultValue: false));
     properties.add(new IntProperty('maxLines', maxLines, defaultValue: 1));
     properties.add(new IntProperty('maxLength', maxLength, defaultValue: null));
-    properties.add(new FlagProperty('maxLengthEnforced', value: maxLengthEnforced, ifTrue: 'max length enforced'));
+    properties.add(new FlagProperty('maxLengthEnforced',
+        value: maxLengthEnforced, ifTrue: 'max length enforced'));
```

Comment:
please leave this as formatted before; the new formatting here is much less readable